src/Application.php

```diff
@@ -113,7 +133,7 @@ protected function runWriteTable(array $tableConfig): void
         }
     }
 
-    public function writeIncremental(CsvFile $csv, array $tableConfig): void
+    public function writeIncremental(SplFileInfo $csv, array $tableConfig): void
```

Comment:
`CsvFile` v predchadzajucej verzii rozsirovalo `SplFileInfo`, a malo metodu `getPathname` ... novy `CsvReader`/`CsvWriter` tuto metodu uz nema, ... okrem toho sa v kode writera casto `CsvReader` ani nepouzije, ak ma DB funkciu vytvorit tabulku priamo z CSV. ... preto je to tu a na dalsich miestach nahradene `SplFileInfo`.